src/sql/workbench/contrib/notebook/test/browser/notebookEditor.test.ts

```diff
@@ -44,7 +52,34 @@ import { IWorkbenchThemeService } from 'vs/workbench/services/themes/common/work
 import { UntitledTextEditorInput } from 'vs/workbench/services/untitled/common/untitledTextEditorInput';
 import { IUntitledTextEditorService } from 'vs/workbench/services/untitled/common/untitledTextEditorService';
 import { workbenchInstantiationService } from 'vs/workbench/test/browser/workbenchTestServices';
+import TypeMoq = require('typemoq');
```

Comment:
`import * as TypeMoq from 'typemoq';`

Reply:
@ranasaria In case you missed it this is the primary thing I'd like you to fix before signing off. We shouldn't be using require in our typescript code.

Reply:
fixed. Will push the change up tomorrow